pkg/sonnet/sonnet.go

```diff
@@ -0,0 +1,36 @@
+package sonnet
```

Comment:
I would use `pkg/jsonnet/jsonnet.go`. I think jsonnet is the right thing to call this.